src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertsArtworks.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+  Text,
+  Flex,
+  Box,
+  Spacer,
+  Join,
+  Separator,
+  Button,
+  GridColumns,
+  Column,
+  SkeletonText,
+  Skeleton,
+  SkeletonBox,
+  Message,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { SavedSearchAlertsArtworksQuery } from "__generated__/SavedSearchAlertsArtworksQuery.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { EditAlertEntity } from "Apps/Settings/Routes/SavedSearchAlerts/types"
+import { SearchCriteriaAttributes } from "Components/SavedSearchAlert/types"
+import { getAllowedSearchCriteria } from "Components/SavedSearchAlert/Utils/savedSearchCriteria"
+import { AlertProvider } from "Components/Alert/AlertProvider"
+import ArtworkGrid, {
+  ArtworkGridPlaceholder,
+} from "Components/ArtworkGrid/ArtworkGrid"
+import { Modal } from "Components/Alert/Components/Modal/Modal"
+import { Media } from "Utils/Responsive"
+import {
+  CriteriaPills,
+  CriteriaPillsPlaceholder,
+} from "Components/Alert/Components/CriteriaPills"
+import { ModalHeader } from "Components/Alert/Components/Modal/ModalHeader"
+
+interface AlertArtworksProps {
+  alert: NonNullable<SavedSearchAlertsArtworksQuery["response"]["me"]>["alert"]
+  onCloseClick?: () => void
+  onEditAlertClick: () => void
+}
+
+export const AlertArtworks: React.FC<AlertArtworksProps> = ({
+  alert,
+  onCloseClick,
+  onEditAlertClick,
+}) => {
+  if (!alert || !alert.artworksConnection)
+    return <SavedSearchAlertsArtworksPlaseholder onCloseClick={onCloseClick} />
+
+  const count = alert.artworksConnection?.counts?.total
+  const href = alert.href
+
+  return (
+    <>
+      <Media greaterThanOrEqual="md">
+        <Box p={4} flex={1}>
+          <Join separator={<Spacer y={2} />}>
+            <ModalHeader />
+
+            <Flex flexWrap="wrap" gap={1}>
+              <CriteriaPills editable={false} />
+            </Flex>
+
+            <Separator />
+
+            {count > 0 && (
+              <Text variant="sm-display" color="black60" mb={4}>
+                {count}
+                {count > 1 ? " works" : " work"} currently on Artsy match your
+                criteria.
+                <Text variant="sm-display" color="black60">
+                  See our top picks for you:
+                </Text>
+              </Text>
+            )}
+
+            <Spacer y={2} />
+
+            <ArtworkGrid
+              artworks={alert.artworksConnection}
+              columnCount={2}
+              emptyStateComponent={<ArtworksGridEmptyState />}
```

Comment:
(non-blocking): What do you think about refactoring this to `count > 0 ? ... text and artwork grid ... : <ArtworksGridEmptyState />`? Like this, we would reduce the number of `count > 0` within the code and would not implicitly return <ArtworksGridEmptyState /> within `ArtworkGrid`.